enterprise/signals.py

```diff
@@ -84,7 +84,8 @@ def _complete_user_enrollment():
                     enrollment.course_id,
                     enrollment.course_mode,
                     cohort=enrollment.cohort_name,
-                    source_slug=getattr(enrollment.source, 'slug', None)
+                    source_slug=getattr(enrollment.source, 'slug', None),
+                    discount_percentage=enrollment.discount_percentage
```

Comment:
nit: it is good to have comma `'` after the last param too if params are in multiline so that when someone else tries to add another param he/she does not have to change your line.